### PR TITLE
💄Book a Demo Modal

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per Levi's email **Re: Tina.io - Meet with Scott requires update**

Hi Josh,
⚠️ IMPORTANT please give this priority as Scott is on leave with no signal. i.e. we might miss a demo because it’s in Scott’s calendar.
 
Here is my booking link: [https://meetings-eu1.hubspot.com/levi-jackson](https://aus01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmeetings-eu1.hubspot.com%2Flevi-jackson&data=05%7C02%7CJoshBerman%40ssw.com.au%7Cb0182aba987145e0971008dc906664ff%7Cac2f7c34b93548e9abdc11e5d4fcb2b0%7C0%7C0%7C638544016674491533%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=5vZmfttkNLFYKAy8EY9gGSRjL%2Fz1Zg00c0ZE4j1K9Uk%3D&reserved=0)
 
1. Easy fix - Please investigate how Scott’s HubSpot booking link works on the website and change the URL from Scott’s to mine
 
2. A better fix – Remember that most customers are from America and so they generally would want to arrange a meeting within their work hours (aka with Scott).
Therefore, it would be great to give the users a choice based on the timezone. i.e. be able to select either Scott, Jean or I based on location/timezone.

We have gone for `2`, a _better fix_:
- Added a modal for booking options when users click 'book a demo' 
- Added links to hubspots for Scott and Levi **❗AWAITING JEANS**

_✅all test passed by @bettybondoc_

Screenshot 2024-06-20 at 11 53 14 am
**Figure: Laptop L view of new modal**

### CODE REVIEW

**Comment by joshbermanssw** [comment]
⚠️ waiting for jean hubspot link

File: components/forms/BookDemo.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        </div>
+        <div className="grid lg:grid-cols-2 gap-3 px-6 md:px-0 lg:px-6">
+          <div className="flex justify-center w-full items-center h-full">
+            <div className="w-full max-w-sm h-full">
+              <Link
+                href="https://meetings-eu1.hubspot.com/scott15?uuid=2f12f1a3-8b6a-4472-bfb5-2f3f930a1b19"
+                className="flex flex-col md:flex-row lg:flex-row w-full h-full items-center justify-between rounded-lg border border-input bg-background p-4 shadow transition transform duration-200 hover:scale-105 hover:bg-muted focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring text-blue-800 hover:text-blue-700"
+                prefetch={false}
+              >
+                <img
```
Comment (JackDevAU) [inline components/forms/BookDemo.tsx:25]: use `next/image` `<Image />` component instead of normal ``

File: components/forms/BookDemo.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    Meet with Scott (English)
+                  </div>
+                </div>
+                <div className="flex-shrink-0">
+                  <ChevronRightIcon className="h-6 w-6 text-muted-foreground" />
+                </div>
+              </Link>
+            </div>
+          </div>
+          {/* <div className="flex justify-center w-full items-center h-full">
```
Comment (JackDevAU) [inline components/forms/BookDemo.tsx:42]: Can this be removed?

File: components/forms/BookDemo.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            </div>
+          </div> */}
+          <div className="flex justify-center w-full items-center h-full">
+            <div className="w-full max-w-sm h-full">
+              <Link
+                href="https://meetings-eu1.hubspot.com/levi-jackson?uuid=2eb9cf82-25fe-40b6-86c3-466c365d34a5"
+                className="flex flex-col md:flex-row lg:flex-row w-full h-full items-center justify-between rounded-lg border border-input bg-background p-4 shadow transition transform duration-200 hover:scale-105 hover:bg-muted focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring text-blue-800 hover:text-blue-700"
+                prefetch={false}
+              >
+                <img
```
Comment (JackDevAU) [inline components/forms/BookDemo.tsx:73]: Same as above use `next/iamge`

File: components/forms/BookDemo.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+          </div>
+        </div>
+      </div>
+    </>
+  );
+};
+
+export default DemoForm;
+
+function ChevronRightIcon(props) {
```
Comment (JackDevAU) [inline components/forms/BookDemo.tsx:98]: Believe this uses `react/icons` can we use the icon from there instead

File: components/layout/Navbar.tsx [hunk 5]
```diff
@@ -166,9 +168,9 @@ export function Navbar({}) {
             <ul className="flex gap-6 min-[1135px]:gap-8 min-[1135px]:gap-12 relative z-20">
               {data.map((item) => {
                 const navLinkClasses =
```
Comment (JackDevAU) [inline components/layout/Navbar.tsx:170]: Do we need a variable for this?

File: components/layout/Navbar.tsx [hunk 6]
```diff
@@ -76,9 +80,9 @@ export function Navbar({}) {
               </li>
               {data.map((item) => {
                 const navLinkClasses =
```
Comment (JackDevAU) [inline components/layout/Navbar.tsx:82]: Do we need a variable for this?

File: components/forms/BookDemo.tsx (hunk 3 above)
Comment (joshbermanssw) [reply components/forms/BookDemo.tsx:73]: ✅

File: components/forms/BookDemo.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/forms/BookDemo.tsx:25]: ✅

File: components/forms/BookDemo.tsx (hunk 4 above)
Comment (joshbermanssw) [reply components/forms/BookDemo.tsx:98]: ✅

File: components/layout/Navbar.tsx (hunk 6 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:82]: Was not apart of my implementation (i.e was there prior - i can move it out perhaps? I think its fine where it is

File: components/layout/Navbar.tsx (hunk 5 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:170]: same as below

File: components/layout/Navbar.tsx (hunk 6 above)
Comment (JackDevAU) [reply components/layout/Navbar.tsx:82]: yea seems weird - lets use it in classnames, move it to a component or use 1 variable at the top. Cause they seem to be the same

File: components/forms/BookDemo.tsx (hunk 2 above)
Comment (joshbermanssw) [reply components/forms/BookDemo.tsx:45]: It has been re-added

File: components/layout/Navbar.tsx (hunk 6 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:82]: ✅

File: components/layout/Navbar.tsx (hunk 5 above)
Comment (joshbermanssw) [reply components/layout/Navbar.tsx:170]: ✅

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**